frontend/views/containers/proposals/AddMembers.vue

```diff
@@ -61,33 +86,144 @@ export default {
     }
   },
   validations: {
-    form: {
-      member: {
-        required
-      }
-    },
     // validation groups by route name for steps
     steps: {
       Member: [
         'form.member'
       ]
     }
   },
+  computed: {
+    ...mapState([
+      'currentGroupId',
+      'loggedIn'
+    ]),
+    ...mapGetters([
+      'currentGroupState',
+      'groupSettings',
+      'groupMembersCount',
+      'groupShouldPropose'
+    ])
+  },
   methods: {
-    submit (form) {
-      console.log(
-        'TODO: Logic to Propose Mincome.',
-        'mincome:', this.form.member,
-        'reason:', form.reason
-      )
+    async addInvitee (index) {
+      const searchUser = this.form.invitees[index]
+
+      if (searchUser === this.$store.state.loggedIn.username) {
+        return this.setInviteError(index, this.L('You can\'t invite yourself'))
+      }
+
+      if (this.currentGroupState.profiles[searchUser]) {
+        return this.setInviteError(index,
+          this.L('{searchUser} is already on the group', { searchUser })
+        )
+      }
+
+      try {
+        const contractID = await sbp('namespace/lookup', searchUser)
+        const userState = await sbp('state/latestContractState', contractID)
+        this.ephemeral.invitesToSend[index] = { contractID, ...userState }
+        Vue.set(this.form.eachFeedbackMsg, index, {
+          text: this.L('Ready to be invited!'),
+          class: 'has-text-success'
+        })
+        this.ephemeral.isValid = true
+      } catch (err) {
+        console.log(err)
+        return this.setInviteError(index, this.L('Non existing user. TODO support general invites'))
```

Comment:
Why is this function `return`ing something?

Reply:
No need in this case. removed